app/src/main/java/ch/epfl/sdp/drone3d/service/impl/drone/DronePhotosImpl.kt

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2021  Drone3D-Team
+ * The license can be found in LICENSE at root of the repository
+ */
+
+package ch.epfl.sdp.drone3d.service.impl.drone
+
+import android.annotation.SuppressLint
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import ch.epfl.sdp.drone3d.service.api.drone.DronePhotos
+import ch.epfl.sdp.drone3d.service.api.drone.DroneService
+import io.mavsdk.camera.Camera
+import io.mavsdk.camera.CameraProto
+import io.reactivex.Single
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+import timber.log.Timber
+import java.io.IOException
+import java.net.MalformedURLException
+import java.net.URL
+import javax.inject.Inject
+
+class DronePhotosImpl @Inject constructor(val service: DroneService) : DronePhotos {
```

Comment:
A small documentation about this class implementation would be appreciated.

Reply:
Done